src/components/Variant.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import FlagContext from "./FlagContext";
+
+class Variant extends Component {
+  render() {
+    const { children, name, isDefault } = this.props;
+    return (
+      <FlagContext.Consumer>
+        {flagValue =>
+          (flagValue === name || (flagValue === null && isDefault)) && children
```

Comment:
so a variant can have a name and still be `isDefault`? if so, good idea

Reply:
correct